plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -115,7 +115,7 @@ func Run(s *options.SchedulerServer) error {
 		glog.Fatal(server.ListenAndServe())
 	}()
 
-	configFactory := factory.NewConfigFactory(kubeClient, s.SchedulerName, s.HardPodAffinitySymmetricWeight, s.FailureDomains)
+	configFactory := factory.NewConfigFactory(leaderElectionClient, s.SchedulerName, s.HardPodAffinitySymmetricWeight, s.FailureDomains)
```

Comment:
is it a typo? do you want to remove client on https://github.com/kubernetes/kubernetes/pull/34084/files#diff-c2f9e998f6bc0386be67c0fa4a9db7d5R91?